packages/arb-token-bridge-ui/src/components/TransactionHistory/PendingClaimsInfo.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+  A small info banner that we show when the user has some pending claims.
+  Format: "You have [X] withdrawals ready to claim. [CTA]" 
+*/
+
+import { CheckCircleIcon } from '@heroicons/react/outline'
+import { OutgoingMessageState } from 'token-bridge-sdk'
+import { useNetworksAndSigners } from '../../hooks/useNetworksAndSigners'
+import { useAppState } from '../../state'
+import { isPending, outgoungStateToString } from '../../state/app/utils'
+import { isFathomNetworkName, trackEvent } from '../../util/AnalyticsUtils'
+import { getNetworkName } from '../../util/networks'
+import { useAppContextDispatch } from '../App/AppContext'
+import { ExternalLink } from '../common/ExternalLink'
+
+export const PendingClaimsInfo = () => {
+  const {
+    l2: { network: l2Network }
+  } = useNetworksAndSigners()
+
+  const {
+    app: { mergedTransactions }
+  } = useAppState()
+
+  const l2NetworkName = getNetworkName(l2Network.chainID)
+
+  const dispatch = useAppContextDispatch()
+
+  const numPendingClaimTransactions = mergedTransactions.filter(
+    tx =>
+      isPending(tx) &&
+      tx.status === outgoungStateToString[OutgoingMessageState.CONFIRMED]
+  ).length
+
+  // don't show this if user doesn't have anything to claim
+  if (!numPendingClaimTransactions) return null
```

Comment:
nit: be explicit about the check `numPendingClaimTransactions === 0`

Reply:
Done